README.md

```diff
@@ -153,6 +153,13 @@ Follow us on Twitter for the latest on Talos, or join our slack for in-depth dis
   </a>
 </p>
 
+Join our weekly Zoom meetings:
+
+https://zoom.us/j/3595189922
```

Comment:
no cute zoom cname?

Reply:
Ha, not yet. Wasn't sure if we should.

Reply:
I wonder if I missed today's meeting? I never saw the link... :confused:

Reply:
That was my fault... I posted in the community channel but not the engineering channel.